documentation/modules/mtv-settings.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="mtv-settings_{context}"]
+= Configuring MTV settings
+
+If you have Administrator privileges, you can access the *Overview* page and change the following settings in it:
+
+[cols="1,1,1",options="header"]
+.{project-short} settings
```

Comment:
this list aligns with the order the items appear in the UI which is good
but @yaacov, we should place the settings that are related to warm migration at the bottom - we consider dropping warm migration at some point...

Reply:
added an issue:
https://github.com/kubev2v/forklift-console-plugin/issues/706

this issues, 705 and 706, are easy to solve, we can update the docs now, or wait until the fixes are merged ...

Reply:
yeah, I'd wait for 706 but we can be explicit about the controller's main container before the fix for 705 gets in

Reply:
I see the order was updated, resolving